deployments/qat_dpdk_app/base/crypto-perf-dpdk-pod-requesting-qat.yaml

```diff
@@ -7,8 +12,8 @@ spec:
   - name: crypto-perf
     image: intel/crypto-perf:devel
     imagePullPolicy: IfNotPresent
-    command: [ "/bin/bash", "-c", "--" ]
-    args: [ "while true; do sleep 300000; done;" ]
+    command: [ "/bin/bash", "-c", "/usr/bin/run-dpdk-test" ]
+    args: [ "--ptest throughput --devtype crypto_qat --optype cipher-only --cipher-algo aes-cbc --cipher-op encrypt --cipher-key-sz 16 --total-ops 10000000 --burst-sz 32 --buffer-sz 64" ]
```

Comment:
this won't work. we might still want to keep the possibility to run crypto/compress-perf tests with different arguments using the env variables as defined by `run-dpdk-test` wrapper. Add another `configMapGenerator` for these using examples given in https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#configmapgenerator

Reply:
I knew it would not work..
My change is that we just put env variables to each case (crypto - TESTCMD:crypto PTEST: <something>, compress - TESTCMD:compress PTEST: <something for dc>) would be enough.